src/modules/dashboard/components/CreatePaymentDialog/CreatePaymentDialogForm.css

```diff
@@ -0,0 +1,56 @@
+.tokenAmount {
+  display: grid;
+  width: 100%;
+  grid-template-columns: auto 100px;
+}
+
+.tokenAmountSelect {
+  align-self: center;
+  margin-left: 20px;
+}
+
+.tokenAmountUsd {
+  justify-self: right;
+}
+
+.domainPotBalance {
+  font-size: var(--size-tiny);
+  font-weight: var(--weight-bold);
+  text-align: right;
+  color: var(--temp-grey-blue-7);
+}
+
+.title {
+  font-size: var(--size-medium-l);
+  font-weight: var(--weight-bold);
+  line-height: 24px;
+  color: var(--dark);
+}
+
+.tokenAmount label,
+.domainSelects label,
+.textAreaSection label {
+  font-size: var(--size-smallish);
+  color: var(--dark);
+}
+
+.tokenAmount input {
+  font-weight: var(--weight-normal);
+  color: var(--dark);
+}
+
+.singleUserContainer label {
+  margin-bottom: 15px;
+  color: var(--dark);
+}
+
+.singleUserContainer > div {
+  padding-left: 0;
+}
+
+.noPermissionFromMessage {
+  font-size: var(--size-tiny);
+  font-weight: var(--weight-bold);
+  line-height: 18px;
+  color: var(--danger);
+}
```

Comment:
The labels are not properly aligned with the error text. Please make sure they are on the same line

![Screenshot from 2020-12-04 12-44-38](https://user-images.githubusercontent.com/1193222/101156212-ed26b880-3630-11eb-8f3c-33bb6d7e241b.png)